web/bulk_rendering/build.sh

```diff
@@ -1,7 +1,14 @@
 #! /bin/bash
 # 
 # Compile the KeymanWeb bulk-renderer module for use with developing/running engine tests.
-#
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+KEYMAN_ROOT="$(dirname "$THIS_SCRIPT")/../.."
```

Comment:
I'd prefer `KEYMAN_ROOT` to only be set by `build-utils.sh` -- even though this looks slightly less clean for the build-utils.sh call: this just helps to validate that this variable is 'owned' by `build-utils.sh`. After calling `build-utils.sh`, the subsequent lines should be able to call it without issue. Technically, we should have the `shellHelperFunctions.sh` include outside the `STANDARD BUILD SCRIPT INCLUDE` section... but that I'm less fussy about.